cabal.project

```diff
@@ -29,6 +29,8 @@ benchmarks: True
 -- Luckily, bloomfilter is not commonly used package, so this is good enough.
 constraints: bloomfilter <0
 
+allow-newer: data-elevator:base
```

Comment:
No longer necessary now that a new data-elevator is out https://github.com/sgraf812/data-elevator/pull/3 (including the `COMPLETE` pragma to avoid the pattern match warning).